scaladoc/src/dotty/tools/scaladoc/ScaladocSettings.scala

```diff
@@ -111,5 +111,8 @@ class ScaladocSettings extends SettingGroup with AllScalaSettings:
   val snippetCompilerDebug: Setting[Boolean] =
     BooleanSetting("-Ysnippet-compiler-debug", snippets.SnippetCompilerArgs.debugUsage, false)
 
+  val generateInkuire: Setting[Boolean] =
+    BooleanSetting("-generate-inkuire", "Generates InkuireDB and enables Hoogle-like searches", false)
```

Comment:
We should call it `-Ygenerate-inkuire` to put emphasis that this flag is private and should not be used without consideration.

Reply:
done